apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java

```diff
@@ -143,7 +143,8 @@ public ItemDTO update(@PathVariable("appId") String appId,
 
     Item beforeUpdateItem = BeanUtils.transform(Item.class, managedEntity);
 
-    //protect. only value,comment,lastModifiedBy can be modified
+    //protect. only value,type,comment,lastModifiedBy can be modified
+    managedEntity.setType(entity.getType());
```

Comment:
I think `com.ctrip.framework.apollo.biz.service.ItemSetService#doUpdateItems` should also be updated.